Formula/homebank.rb

```diff
@@ -2,9 +2,9 @@ class Homebank < Formula
   desc "Manage your personal accounts at home"
   homepage "http://homebank.free.fr"
   # A mirror is used as primary URL because the official one is unstable.
-  url "https://ftp.openbsd.org/pub/OpenBSD/distfiles/homebank-5.6.3.tar.gz"
-  mirror "http://homebank.free.fr/public/sources/homebank-5.6.3.tar.gz"
-  sha256 "419475f564bbd9be7f4101b1197ce53ea21e8374bcf0505391406317ed823828"
+  url "http://homebank.free.fr/public/sources/homebank-5.6.5.tar.gz", using: :homebrew_curl
+  mirror "https://ftp.openbsd.org/pub/OpenBSD/distfiles/homebank-5.6.5.tar.gz"
+  sha256 "b5494dfcf87d7a8572696a2afa2a04496ba79e3857cd3e3b18ac03bd1b6d2ca8"
```

Comment:
Homepage URL may have some sort of rate limiter given it is redirected to overload.html. This may make it more troublesome in Homebrew's CI environment.

OpenBSD mirror is dependent on the OpenBSD maintainer. Not something we or the upstream have control over. Looks like they are about 1 version behind and for some reason it isn't on their `ftp.openbsd.org` server

Could try to find the most actively updated repository (https://repology.org/project/homebank/versions) that also mirrors tarball, but our livecheck will always be faster than the mirror copy. Options:
- MacPorts - https://distfiles.macports.org//homebank/homebank-5.6.5.tar.gz
- Debian - http://deb.debian.org/debian/pool/main/h/homebank/homebank_5.6.5.orig.tar.gz
- FreeBSD - https://pkg.freebsd.org/ports-distfiles/homebank-5.6.5.tar.gz

Would be nice to have an official mirror but I don't think any are available.

Reply:
yeah, probably we have to switch to use those mirrors. Let me update the PR for it.

Reply:
The openbsd is not up to date.